go-controller/pkg/config/config.go

```diff
@@ -167,6 +167,12 @@ type DefaultConfig struct {
 	// ClusterSubnets holds parsed cluster subnet entries and may be used
 	// outside the config module.
 	ClusterSubnets []CIDRNetworkEntry
+	// NetFlowTargets holds NetFlow targets for exporting flow data.
```

Comment:
@rcarrillocruz should this be in another section than the [main]/[default]/etc? They're logically grouped, but not really related to the other stuff here. And we may have other related options in the future.

Reply:
Also, is this something we hsould be able to enable/disable at runtime, or will it just always be active?

Reply:
What do you suggest? New 'ExportFlows' sounds good?

Reply:
k, let's rename it 'Monitoring' for future proofing it

Reply:
FWIW we settled on a new [monitoring] group via Slack.

Reply:
done

Reply:
And yes, it can be disabled too at runtime.

To enable:

kubectl -n ovn-kubernetes set env ds/ovnkube-node -c ovnkube-node OVN_NETFLOW_TARGETS=192.168.1.1:2048,192.168.1.2:2055

To disable:

Unset the env var.